src/index.html

```diff
@@ -22,6 +22,378 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__logo">
+        <a href="/">
+          <img
+            src="./images/logo.png"
+            alt="Moyo Logo"
+          />
+        </a>
+      </div>
+      <nav class="navigation">
+        <ul class="menu">
+          <li class="menu__item">
+            <a
+              href="#"
+              class="menu__link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="menu__item">
```

Comment:
```suggestion
          </li>

          <li class="menu__item">
```
It's easier to reed code with blank line between **sibling** elements